libraries/GCS_MAVLink/GCS.h

```diff
@@ -287,6 +287,12 @@ class GCS_MAVLINK
 
     virtual uint64_t capabilities() const;
 
+    // set an inverval for a specific mavlink message.  Returns false
```

Comment:
This change (making this method public) doesn't seem necessary.  It's not necessarily a bad thing, but I don't think we need it for this PR, and the longer we keep stuff non-public the easier to update/fix things.

Reply:
hmm... looked like it was. My bad. I'll revert that part

Reply:
fixed